components/organisms/Drawer.js

```diff
@@ -0,0 +1,82 @@
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import { useTranslation } from "next-i18next";
+import FocusTrap from "focus-trap-react";
+import { ActionButton } from "../atoms/ActionButton";
+
+export const Drawer = ({ children }) => {
+  const [isOpen, setisOpen] = useState(false);
+  const { t } = useTranslation("common");
+
+  const toggleDrawer = () => {
+    setisOpen(!isOpen);
+    const drawerEl = document.getElementById("drawer");
```

Comment:
Perhaps there is a way to pass in a prop to change the style of these components similar to action button?